lib/terraform/command_options.rb

```diff
@@ -56,6 +56,14 @@ def state=(value)
       store key: 'state', value: value
     end
 
+    def state_out
+      fetch key: 'state-out'
+    end
+
+    def state_out=(value)
```

Comment:
The `=` is a Ruby convention for setter methods. More importantly, though, is that I plan to remove this class soon so we can just leave it as matching the options for now.